ddp-workspace/projects/ddp-testboston/src/app/user-registration-prequal/userRegistrationPrequal.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, Inject } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ToolkitConfigurationService } from 'toolkit';
+import { InvitationServiceAgent } from '../../../../ddp-sdk/src/lib/services/serviceAgents/invitationServiceAgent.service';
+import { Auth0AdapterService } from 'ddp-sdk';
+import { ErrorType } from '../models/errorType';
+import { take } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-user-registration-prequal',
+  template: `
+      <div *ngIf="errorMessage" class="ErrorMessage">
+        <span>{{ errorMessage }}</span>
+      </div>
+      <form class="registration-prequal-form" [formGroup]="formGroup" (ngSubmit)="onSubmit()">
+        <mat-form-field class="example-input-field" floatLabel="always">
+          <mat-label>Enter invitation code</mat-label>
+          <input matInput
+                 uppercase
+                 formControlName="invitationId"
+                 pattern="[a-zA-Z0-9]{12}"
+                 maxLength="12"
+                 placeholder="Your invitation code">
+        </mat-form-field>
+        <mat-form-field class="example-input-field" floatLabel="always">
+          <mat-label>What is your current mailing zip code?</mat-label>
```

Comment:
Need to use plain material widget but tried to pickup whatever styling class we were using.